docs/process/guidance/feature_flags/index.rst

```diff
@@ -0,0 +1,507 @@
+   # terms of the Apache License Version 2.0 which is available at
+   # https://www.apache.org/licenses/LICENSE-2.0
+   #
+   # SPDX-License-Identifier: Apache-2.0
+   # *******************************************************************************
+
+Feature Flags
+=============
+
+Background
+----------
+
+Feature flags
+^^^^^^^^^^^^^
+
+Feature flags, also known as feature toggles, are a technique used in software
+development to enable or disable features dynamically. The main goal is to
+allow teams to manage and release new features more effectively, enabling A/B
+testing and quickly roll back if issues arise. This guidance aims
+to define how we intend to make use of these features for configuration and
+variant management of SCORE.
+
+There are two categories of flags, build flags and runtime feature toggles:
+
+- **Build Flags**: These are build/compile time switches used to include or
+  exclude certain pieces of code from the build process. When a build flag is
+  set, the corresponding code is compiled into the final application/consumer.
+  These flags can be used for specific environments, such as build vs. debug.
+  Build flags typically require a new full deployment to change the state of the
+  feature.
+
+- **Runtime Feature toggles**: These toggles control features at runtime,
+  meaning they can be changed with partial deployments or ultimately while the
+  application is running. They are generally more flexible, allowing for easier
+  experimentation. Runtime toggles are typically implemented through
+  configuration settings in the application or by querying an external service
+  for configuration.
+
+This guidance will focus on the former.
+
+Example for a feature flag
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Feature flags are defined in BUILD Files:
+
+.. code-block:: bazel
+    :caption: some_module/BUILD
+
+    load("@bazel_skylib//rules:common_settings.bzl", "bool_flag")
+
+    bool_flag(
+        name = "some_feature",
+        build_setting_default = True,
+    )
+
+    config_setting(
+        name = "config_some_feature",
+        flag_values = {
+            ":some_feature": "True",
+        },
+    )
+
+Feature flags can be specified via the command line to quickly enable or
+disable features for builds and tests.
+
+.. code-block:: bash
+
+    bazel build \
+    --platforms=//platform:some_platform \ # TODO: To be defined with overall integration
+    --//some_module:some_feature=True \
+    --extra_toolchains=//toolchains:some_compiler //some_module:some_target
+
+
+Defining convenient combinations of feature flags
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To simplify the user interface, configurations can be defined that specify
+values for a set of feature flags.
+
+.. code-block:: bash
+    :caption: .bazelrc
+
+    build:feature_flag_values_non_default --//some_module:some_feature=False --//yet_another_module:some_other_feature=False
```

Comment:
Can you provide a more understandable example?
It doesn't need to match score scope. Rather something people will understand. Something trivial like "no rear doors = disable left rear door and disable right rear door".

Reply:
Sure. I'll just take your door example 😉